docs/01-app/03-api-reference/04-functions/cacheLife.mdx

```diff
@@ -37,6 +37,12 @@ const nextConfig = {
 export default nextConfig
 ```
 
+> **Good to know**:
```

Comment:
This feels like required knowledge, not something worth knowing. If you do not have 'use cache' cacheLife wont work at all